src/jit/compiler.hpp

```diff
@@ -3127,6 +3127,16 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 */
 
+//------------------------------------------------------------------------
+// compUpdateLife: Update the GC's masks, register's masks and reports change on variable's homes given a set of
+//    current live variables if changes have happened since "compCurLife".
```

Comment:
grammar suggestion =>
`Update the GC and register masks, and report changes of variable homes given a set of
current live variables. The new liveness information is compared against the existing liveness
in "compCurLife", and changes are only reported if there are differences.`

Reply:
That sounds clearer, thanks!

Reply:
I have done the same as before, so I will remove the comment on the header file for this function too.